master/buildbot/www/oauth2.py

```diff
@@ -237,6 +237,9 @@ def __init__(self,
             self.tokenUri = '{0}/login/oauth/access_token'.format(serverURL)
         self.serverURL = serverURL or self.resourceEndpoint
 
+        enterpriseGraphqlEndpoint = None
```

Comment:
Please do the lstrip() line 231 (overlook of a previous PR)

Please simplify the code to avoid two `if serverURL is not None` lines.

Reply:
oh okay, did an lstrip but probably misunderstood the ask. sorry about that, on it